packit_service/service/api/allowlist.py

```diff
@@ -0,0 +1,32 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+from http import HTTPStatus
+from logging import getLogger
+
+from flask_restx import Namespace, Resource
+
+from packit_service.models import AllowlistModel
+
+logger = getLogger("packit_service")
+
+ns = Namespace("allowlist", description="Allowlisted FAS accounts")
+
+
+@ns.route("")
+class Allowlist(Resource):
```

Comment:
Just to be sure. Do we use it somewhere?

Reply:
I'm not sure tbh, internally we don't need to use API I guess, permitting/denying happens also happens in the cluster. Informative value maybe?

Reply:
We use API from the dashboard but I am not sure if it is used there. Can you please check that?

Reply:
Approving, please merge once this is checked.

Reply:
haven't found any reference from dashboard